api-tests/apply_route_test.go

```diff
@@ -600,6 +612,7 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		{"3.6-latest", "1.7.0", nil, "3.6.21-10.0"},
 		{"3.6-latest", "1.6.0", nil, "3.6.21-10.0"},
 		{"3.6-latest", "1.5.0", nil, "3.6.19-7.0"},
+		{"7.0-recommended", "1.18.0", nil, "7.0.14-8"},
 		{"7.0-recommended", "1.17.0", nil, "7.0.12-7"},
 		{"7.0-recommended", "1.16.2", nil, "7.0.8-5"},
 		{"7.0-recommended", "1.16.1", nil, "7.0.8-5"},
```

Comment:
Please add "5.0-recommended", "1.18.0" & "5.0-recommended", "1.18.0"  and missed recommended for lower operators

Reply:
you mean "5.0-recommended", "1.18.0" & "6.0-recommended", "1.18.0" ?

Reply:
Yes, we have {"6.0-recommended", "1.17.0", nil, "6.0.16-13"}, but do not have {"6.0-recommended", "1.18.0", nil, "6.0.XX-XX"} , the same for 5.0
 https://github.com/Percona-Lab/percona-version-service/pull/323/files#diff-b44f45d7bc5df6be977bcd179a3daf56dcc1c0c9dc2d611f8720b8aa233faf0aR620

Reply:
> and missed recommended for lower operators
I will do that in a separate PR, after this one is merged, because I want to keep this one as a baseline for future reference and I don't want to make it confusing.